src/components/Uploader/index.js

```diff
@@ -17,35 +18,11 @@ const UploadProcessing = () =>
     <div className={style.processing}>Processing your document</div>
   </div>
 
-const UploadError = ({children}) =>
-  <div className={`${style.text} ${style.error}`}>{children}</div>
+const UploadError = ({error}) =>
+  <div className={`${style.text} ${style.error}`}>{error}</div>
 
-const InvalidCapture = ({message}) =>
-  <UploadError>{message}</UploadError>
-
-InvalidCapture.defaultProps = {
-  message: 'We are unable to detect an identity document in this image. Please try again.'
-}
-
-const InvalidFileType = ({message}) =>
-  <UploadError>{message}</UploadError>
-
-InvalidFileType.defaultProps = {
-  message: 'The file uploaded has an unsupported file type.'
-}
-
-const InvalidFileSize = ({message}) =>
-  <UploadError>{message}</UploadError>
-
-const ServerError = ({message}) =>
-  <UploadError>{message}</UploadError>
-
-InvalidFileSize.defaultProps = {
-  message: 'The file size limit of 10 MB has been exceeded. Please try again.'
-}
-
-ServerError.defaultProps = {
-  message: 'There was an error connecting to the server. Please wait and try again later.'
+const showError = (uploading, error) => {
```

Comment:
this should be a component, no reason for it to be a function. If it's component it's much more idiomatic